apps/sms/services/js/messaging/messaging_client.js

```diff
@@ -0,0 +1,54 @@
+/* global bridge */
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for messaging functionality.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'messaging-service';
+  const TIMEOUT = 5000;
+
+  /**
+   * Reference to active bridge client instance.
+   *
+   * @type {Client}
+   */
+  var client;
+
+  var serviceEndpoint = null;
+
+  var MessagingClient = {
+    init() {
+      serviceEndpoint = new SharedWorker(
+        '/services/js/messaging/messaging_service.js'
+      );
+
+      client = bridge.client({
```

Comment:
Per Willson's cpmment client should be able to accept both original way of parameters and client option below. I think it might be more clear to use options since people will know the parameter immediately.